go/interpreter/lfvm/lfvm_test.go

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package lfvm
+
+import (
+	"bytes"
+	"errors"
+	"testing"
+
+	"github.com/Fantom-foundation/Tosca/go/tosca"
+)
+
+func TestVm_Run(t *testing.T) {
+
+	tests := map[string]struct {
+		code           []byte
+		revision       tosca.Revision
+		expectedResult tosca.Result
+		expectedError  *tosca.ErrUnsupportedRevision
+	}{
+		"empty code": {
+			revision: tosca.R13_Cancun,
+			expectedResult: tosca.Result{
+				Success:   true,
+				GasLeft:   1000000,
+				GasRefund: 0,
+				Output:    []byte{}},
+		},
+		"invalid code": {
+			code:     []byte{0x0C},
```

Comment:
To make the intention clearer, please use the explicitly defined `INVALID` op-code
```suggestion
			code:     []byte{vm.INVALID},
```